ode-wrapper/src/main/java/com/trihydro/odewrapper/controller/WydotTimBaseController.java

```diff
@@ -55,20 +55,22 @@ public abstract class WydotTimBaseController {
     protected WydotTimService wydotTimService;
     protected TimTypeService timTypeService;
     private TimType timType = null;
-    private SetItisCodes setItisCodes;
+    private final SetItisCodes setItisCodes;
     protected ActiveTimService activeTimService;
     protected RestTemplateProvider restTemplateProvider;
     MilepostReduction milepostReduction;
     protected Utility utility;
     protected TimGenerationHelper timGenerationHelper;
+    protected MilepostService milepostService;
 
     protected static Gson gson = new Gson();
     private List<TimType> timTypes;
+    protected final List<Integer> bufferTimITISCodes;
 
     public WydotTimBaseController(BasicConfiguration _basicConfiguration, WydotTimService _wydotTimService,
             TimTypeService _timTypeService, SetItisCodes _setItisCodes, ActiveTimService _activeTimService,
             RestTemplateProvider _restTemplateProvider, MilepostReduction _milepostReduction, Utility _utility,
-            TimGenerationHelper _timGenerationHelper) {
+            TimGenerationHelper _timGenerationHelper, MilepostService _milepostService) {

```

Comment:
Suggestion (non-blocking): Move to conventional java parameter naming here instead of using the _ prefix also.